x-pack/plugins/gis/public/_main.scss

```diff
@@ -34,21 +34,24 @@ map-listing, .gisListingPage {
   flex-grow: 1;
 }
 
-.LayerControl {
+.widgetOverlay {
```

Comment:
If you can, prefix your css with your app.... I'd recommend `.mapWidgetOverlay` or `.gisWidgetOverlay`

Reply:
I will you gis even though the app is called map because gis is more unique and this is not user facing

Reply:
@nreese Are you going to change these soon? I'm almost ready to submit my pr for you